config_data:
  move_group: "sia5"
  world_frame: "base_link"
  ft_frame: "eef"
  control_frame: "base_link"
  init_pos: [0.0, 0.45, 0.0, 0.35, 0.0, 0.80, 0.0]
